src/database/__init__.py

```
@@ -12,13 +12,13 @@
 )
 from database.models.movies import (
     MovieModel,
-    LanguageModel,
-    ActorModel,
+    DirectorModel,
```

Ruff (0.11.9)

`database.models.movies.DirectorModel` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)